test/__mocks__/session.mock.ts

```diff
@@ -1593,6 +1614,141 @@ export const APPLICATION_DATA_CH_REF_UPDATE_MOCK: ApplicationData = {
   [updateType.UpdateKey]: UPDATE_OBJECT_MOCK
 };
 
+export const FETCH_BO_APPLICATION_DATA_MOCK: ApplicationData = {
+  overseas_entity_id: '123',
+  transaction_id: '345',
+  entity_number: '1',
+  update: {
+    review_beneficial_owners_individual: [
+      {
+        id: '9999',
```

Comment:
could the id be "1111" and ch_reference "hashed1111" for example? 
my concern is that the ids are all the same and they should be unique.

Reply:
kept the ch_reference the same as the private_bo_mock has 111 (matching them) but changed id to 1111, 2222, 3333

Reply:
fine